File: CSharp/ProjectEuler/ProjectEuler/Problem5.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectEuler
{
    class Problem5
    {
        public void run()
        {
            int smallest_positive_number = smallest_positive();
            Console.WriteLine("The smalles positive number divided by all numbers\n from 1 to 20 is : " +smallest_positive_number);
        }
        public int smallest_positive()
        {          
            int smallest = 1;
            do {
                smallest++;
            } while (isDivisible(smallest) != true);
            return smallest;
        }
        public bool isDivisible(int number)
        {
            for(int i = 1; i <= 20; i++)
            {
                if (number % i != 0)
                {
                    return false;
                }
            }
            return true;
        }
        
    }
}

File: Java/ProjectEuler/src/PE/Problem5.java
package PE;

/**
 * Created by Daniel on 2015-08-27.
 */
public class Problem5 {
    public static void main(String[] args){
        int number = 0;
        do{

            number++;
            System.out.println(number);
        }while(isDivisible(number)!=true);
        System.out.println(number);
    }
    public static boolean isDivisible(int number){
        for(int i = 1; i<=20;i++){
            if(number%i!=0){return false;}
        }
        return true;
    }
}
